src/styles/typo.ts

```
@@ -130,6 +130,105 @@ const typo = {
+        fontSize: '12px',
+        lineHeight: '1.3',
+        letterSpacing: `${12 * -0.05}px`,
+      },
+      Medium_8: {
+        fontFamily: 'Pretendard',
+        fontSize: '8.5px',
+        lineHeight: '1.3',
+        letterSpacing: `${8 * -0.05}px`,
+      },
+    },
+  },
```

Biome (1.9.4)

[error] This property value named Mobile is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named Mobile


(lint/suspicious/noDuplicateObjectKeys)